### PR TITLE
[insight] add integration deepflow doc

### PR DESCRIPTION
**Pull request by Frapschen**

### CODE REVIEW

File: docs/zh/docs/insight/best-practice/integration_deepflow.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## 前提条件
+
+- Global 集群已经安装 Insight
+- Insight 最低版本要求为 v0.23.0
+
+## 安装 DeepFlow Server
+
+1. 安装 DeepFlow chart
+
+DeepFlow Server 需要安装在 Global 集群中(它会默认安装 DeepFlow Agent)。进入 kpanda-global-cluster 集群，在左侧导航栏内点击
```
Comment (windsonsea) [inline docs/zh/docs/insight/best-practice/integration_deepflow.md:14]:
```suggestion
    DeepFlow Server 需要安装在 Global 集群中(它会默认安装 DeepFlow Agent)。进入 kpanda-global-cluster 集群，在左侧导航栏内点击
```
Need to indent 4 space from line 14 - 30

File: docs/zh/docs/insight/best-practice/integration_deepflow.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+![img.png](./images/deepflow_chart_config.png)
+
+大部分 values 都有默认值。其中 DeepFlow 依赖的 Clickhouse 和 Mysql 都需要申请存储卷，他们的默认大小都是 `10Gi`，可以通过 `persistence`
+关键字搜索到相关配置并修改它们。
+
+配置好后就可以点击确定，执行安装了。
+
+2. 修改 insight 配置
+
+在安装 DeepFlow 后，还需要在 Insight 中开启相关的特性门。在左侧导航栏内点击 `配置与密钥` -> `配置项`， 搜索框查询
```
Comment (windsonsea) [inline docs/zh/docs/insight/best-practice/integration_deepflow.md:34]: Indent 4 space from line 34 - 45

**Review by windsonsea — COMMENTED** [review]
Any list has a requirement to indent 4 spaces for its included items

File: docs/zh/docs/insight/best-practice/integration_deepflow.md (hunk 1 above)
Comment (Frapschen) [reply docs/zh/docs/insight/best-practice/integration_deepflow.md]: done

File: docs/zh/docs/insight/best-practice/integration_deepflow.md (hunk 2 above)
Comment (Frapschen) [reply docs/zh/docs/insight/best-practice/integration_deepflow.md]: done

**Review by windsonsea — APPROVED** [review]

**Pull request merged**